support-frontend/assets/pages/[countryGroupId]/checkout/helpers/stripeCheckoutSession.ts

```diff
@@ -0,0 +1,94 @@
+import { storage } from '@guardian/libs';
+import {
+	boolean,
+	nullish,
+	number,
+	object,
+	optional,
+	picklist,
+	safeParse,
+	string,
+} from 'valibot';
+import { isoCountries } from 'helpers/internationalisation/country';
+import type {
+	FormAddressFields,
+	FormPersonalFields,
+} from './formDataExtractors';
+
+export type PersistableFormFields = {
```

Comment:
Should we derive this type from the schema?

Reply:
Yeah that's a good shout